css/card.css

```diff
@@ -389,6 +389,50 @@ section#gacha-controls fieldset:nth-child(2) > input:first-child {
   display: none;
 }
 
+@media screen and (min-width: 1600px) {
+  .details-dialog-card {
+    /* make the card larger when the screen is resized */
+    --card-width: 300px;
+    --card-height: calc(300px * 1.395);
+  }
+  .details-dialog-card.basic:after, #gacha-controls .card-image.basic:after {
+    background-size: 150% !important;
+    background-repeat: repeat;
+  }
+}
+
+@media screen and (min-width: 2200px) {
+  .details-dialog-card {
+    /* make the card larger when the screen is resized */
+    --card-width: 350px;
+    --card-height: calc(350px * 1.395);
+  }
+  .details-dialog-card.basic:after, #gacha-controls .card-image.basic:after {
+    background-size: 120% !important;
+    background-repeat: repeat;
+  }
+}
+
+@media screen and (min-width: 1600px) and (max-height: 1200px),
+screen and (max-height: 820px) {
+  #card-details-dialog {
+    /* let whole dialog scroll */
+    overflow: auto;
+  }
+
+  .details-dialog-card {
+    /* make the card sit up the top instead */
+    align-self: start;
+  }
+
+  .details-dialog-text {
+    /* Override overflow: auto for small screens, 
```

Comment:
The independent scrolling of the text sections for large dialogs no longer seems to be working - the whole dialog scrolls now and the card no longer remains "locked" in view
If you can't figure it out in time, you can merge - I think this is preferred over a bit of text being unreadable

Reply:
I can't seem to reproduce this sorry. When I try to test this the text is never long enough to be scrollable, but the card is too big; therefore the whole thing ends up scrolling. Maybe the card can be smaller after all? 700px feels a bit uncomfortable to me somehow.

Reply:
don't quite follow your reply - maybe it was in response to the code I highlighted which may have nothing to do with the actual issue I was pointing out 😅 
This is the current dialog without the changes in this PR - no matter where the text is scrolling the card is always on screen
![current dialog](https://user-images.githubusercontent.com/47371080/215016188-283ebff5-259f-448c-86ef-8b826a93e148.png)
In this PR, the card can scroll out of view
![new dialog](https://user-images.githubusercontent.com/47371080/215016217-28eaafdb-042e-492b-8652-430cf1ee124c.png)

Reply:
Yeah I think it's to do with the card height, since you can the top and bottom spacing in the top screenshot are slightly different sizes - I wanted it scroll when the spacing became uneven. But I can see I probably need to be MUCH MORE specific in the set sizing, and it can be done better. I will revisit. :)